language-api/src/main/java/de/jplag/tokentypes/ObjectOrientationTokens.java

```diff
@@ -0,0 +1,22 @@
+package de.jplag.tokentypes;
+
+import de.jplag.TokenType;
+
+public enum ObjectOrientationTokens implements TokenType {
```

Comment:
I would expect the tokens for classes, interfaces, and enums here or maybe in a separate enum for types.